### DISCUSSION TITLE
Outdated Mage_Captcha module

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#2348] [Ideas]
If this module cannot be updated to provide an improved interface (those strings are hard to see on smartphones), but also look primitive for our days, it should be removed from OpenMage.

I have never used it precisely because of the difficult to identify codes. Maybe 14 years ago when Magento 1 was released it was fine, today no way. In addition, in general such checks tend to annoy visitors for spending more time and other safety measures should be taken.

**Comment by addison74** [comment]
This module should be removed from OpenMage and offered as a separate repository as we did with others like Backup, PageCache, Poll. There are a few reCaptcha free module that can be added to OpenMage as replacement. This could be one of them

https://github.com/empiricompany/reCaptcha

**Comment by sreichel** [comment]
Created issue for it.
